scripts/delayed.js

```diff
@@ -17,4 +17,27 @@ function loadAdobeLaunch() {
   });
 }
 
-if (!window.location.host.includes('localhost')) loadAdobeLaunch();
+async function loadIDServlet() {
+  const sessionID = getCookieValue('XSESSIONID');
+  const options = {
+    method: 'POST',
+    body: `sameAs=%7B%22cregcontactid%22%3A%22${sessionID}`,
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+    },
+  };
+  const url = '/bin/bhhs/graphIdServlet';
+  const resp = await fetch(url, options);
+  if (resp.ok) {
+    const id = await resp.json();
+    const idString = JSON.stringify(id);
+    document.cookie = `consumerID=${idString}`;
+  }
+}
+
+if (!window.location.host.includes('localhost')) await loadAdobeLaunch();
+if (!getCookieValue('consumerID')) {
+  loadIDServlet();
+}
+
+loadScript('https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit', { async: true, defer: true });
```

Comment:
Do we need to load this on every page, even if we never show the form?

Reply:
Moved to block.